### PR TITLE
[TAxis] print warning if trying to unzoom without gPad

### PR DESCRIPTION
**Pull request by goi42**
Initially reported in the forum [here](https://root-forum.cern.ch/t/th1-stat-calculations-are-inconsistent/40152), this warns user that `UnZoom` does nothing if the axis hasn't been drawn.

### CODE REVIEW

**Comment by lmoneta** [comment]
@phsft-bot build !

**Comment by oshadura** [comment]
@phsft-bot build!

**Comment by oshadura** [comment]
Failure is unrelated: see https://github.com/root-project/root/pull/6127

**Comment by oshadura** [comment]
@couet can you take a look? Looks like this PR could be ready to go?

**Comment by oshadura** [comment]
@lmoneta this PR looks ready, can you take a look?

File: hist/hist/src/TAxis.cxx [hunk 1]
```diff
@@ -1124,7 +1124,10 @@ void TAxis::Streamer(TBuffer &R__b)
 
 void TAxis::UnZoom()
 {
-   if (!gPad) return;
+   if (!gPad) {
+      Warning("TAxis::UnZoom","Cannot UnZoom if gPad does not exist. Did you mean to draw the TAxis fist?");
```
Comment (lmoneta) [inline hist/hist/src/TAxis.cxx:1128]: fist ---> first

**Comment by goi42** [comment]
@lmoneta ah, yes, thanks, sorry about that. fixed now.

**Comment by lmoneta** [comment]
@phsft-bot build!

**Pull request merged**